coderd/database/queries/workspacebuildparameters.sql

```diff
@@ -14,3 +14,28 @@ FROM
     workspace_build_parameters
 WHERE
     workspace_build_id = $1;
+
+-- name: GetUserWorkspaceBuildParameters :many
+SELECT
+    sub.name,
+    sub.value,
+    sub.created_at
+FROM (
+    SELECT
+        wbp.name,
+        wbp.value,
+        wb.created_at,
+        ROW_NUMBER() OVER (PARTITION BY wbp.name ORDER BY wb.created_at DESC) as rn
```

Comment:
Product question: last used parameter value vs. most used one?

Reply:
recency vs. frequency - an age old question. This another two day door so don't want to hold up the PR if there’s no rapid consensus.

Reply:
I agree, it depends on the customer. The best option would be to ask a real one. Anyway, let's leave it for now.